test.js

```diff
@@ -132,26 +141,27 @@ describe('Persistence', (done) => {
 
   it('Record a vote to a restarted session', (done) => {
     chai.request(app)
-    .post('/actions')
-    .send({
-      payload: JSON.stringify({
-        channel: { id: 1 },
-        actions: [{ value: 'Medium' }],
-        user: { name: 'User 2'},
-        original_message: { text: 'Feature 1' }
+      .post('/actions')
+      .send({
+        payload: JSON.stringify({
+          channel: { id: 1 },
+          actions: [{ value: 'Medium' }],
+          user: { name: 'User 2' },
+          original_message: { text: 'Feature 1' }
+        })
+      })
+      .end((err, res) => {
+        res.should.have.status(200)
+        res.should.be.json
+        res.body.attachments[0].text.should.have.string('2 vote')
+        res.body.attachments[0].text.should.have.string('User 1')
+        res.body.attachments[0].text.should.have.string('User 2')
+
+        done()
       })
-    })
-    .end((err, res) => {
-      res.should.have.status(200)
-      res.should.be.json
-      res.body.attachments[0].text.should.have.string('2 vote')
-      res.body.attachments[0].text.should.have.string('User 1')
-      res.body.attachments[0].text.should.have.string('User 2')
-
-      done()
-    })
   })
 
 })
 
-const pprint = (json) => JSON.stringify(json, null, '\t')
\ No newline at end of file
+// Zsuark - 20170317 - Is this pprint function ever used?
```

Comment:
@Zsuark Actually this is never used.

But don't worry. We can detect and fix this kind of stuff when we'll setup a linter.